src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -16,6 +16,14 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		executeAction(e.getActionCommand());
+	}
+
+	public void executeAction(String actionName) {
+		if (actionName.equals("Figure Joe 1")) {
+			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		} else if (actionName.equals("Figure Joe 2")) {
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.